docs/guides/quickstart.md

```diff
@@ -446,6 +446,62 @@ cause a `getPlanets` to get queued up with the application's repo.
 Microcosm will process the action, sending updates to the domains who
 indicate they want to get updates based on their `register` function.
 
+## Handling user interaction
+
+This is looking great, but most applications respond to user interaction
+in some way, so lets look at one way to handle that in Microcosm.
+
+Microcosm provides a couple of options for capturing user input and
+passing it in to actions. One of these is by using the `ActionButton`
+addon. Let's modify our View layer to pull that in:
+
+```javascript
+import React from 'react'
+import ActionButton from 'microcosm/addons/action-button'
+import {addPlanet} from '../actions/planets'
+
+export default function PlanetList ({ planets = [] }) {
+
+  return (
+    <div>
+      <ul>
+        { planets.map(p => <li key={p}>{p}</li>)}
+      </ul>
+
+      <ActionButton action={addPlanet} value="Alpha Centauri">
+        Add Planet
+      </ActionButton>
+    </div>
+  )
+}
+```
+
+When clicked, our button will broadcast the `addPlanet` action. This
+won't do anything currently because the `addPlanet` action doesn't
+exist. Let's create that action and subscribe our Domain to it:
+
+```javascript
+// add to src/actions/planets.js
+export function addPlanet (planet) {
+  return planet
+}
+```
+
+```javascript
+// update register() method in src/domains/planets.js
+register () {
+  return {
+    [getPlanets]: this.append,
```

Comment:
wouldn't want someone who was following along to remove that line from this method though as it would bork the little app they were building